ocs2_arm_controller:
  type: ocs2_arm_controller/Ocs2ArmController
  
  # Joint configuration
  joints:
    - joint1
    - joint2
    - joint3
    - joint4
    - joint5
    - joint6

  command_interfaces:
    - position
  state_interfaces:
    - position
  
  # Control parameters
  home_pos: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0]
  zero_pos: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0]